toolkit/scripts/pkggen.mk

```diff
@@ -86,6 +86,12 @@ $(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_d
 		--worker-tar $(chroot_worker) \
 		$(if $(filter y,$(RUN_CHECK)),--run-check) \
 		$(logging_command) \
+		--cpu-prof-file=$(PROFILE_DIR)/specreader.cpu.pprof \
```

Comment:
This could probably  be done the same way we define `$(logging_command)` since its all the 'same' for each tool.

Reply:
I tried to do it as generically as possible, but my make skills are not on point, and although it worked, but the resultant filenames were not self evident of the stages.
e.g for the spec reader, it created a file called `specs.trace`, i would prefer `specreader.trace` though.

Should we could keep it as is, what do you think ?
@dmcilvaney

Reply:
While you would still end up with the ugly naming, I think this would work if dropped into `profile.mk`:
```make
PROFILE_DIR    ?= $(BUILD_DIR)/profile
traceing_file   = $(PROFILE_DIR)/$(notdir $@)
tracing_command = --cpu-prof-file=$(traceing_file).cpu.pprof \
                  --mem-prof-file=$(traceing_file).mem.pprof \
                  --trace-file=$(traceing_file).trace \
                  $(if $(filter y,$(ENABLE_CPU_PROFILE)),--enable-cpu-prof) \
                  $(if $(filter y,$(ENABLE_MEM_PROFILE)),--enable-mem-prof) \
                  $(if $(filter y,$(ENABLE_TRACE)),--enable-trace)
```
You could then call it via
```make
	$(go-imageconfigvalidator) \
		--input=$(CONFIG_FILE) \
		--dir=$(CONFIG_BASE_DIR) \
		$(tracing_command ) \
		--timestamp-file=$(TIMESTAMP_DIR)/imageconfigvalidator.jsonl && \
	touch $@
```

Reply:
This is exactly what I did, the names ended up not being pleasing to the idea, or give the intent of what it was.
e.g for the specreader, the file was `specs.trace`

Inn anycase, let me read more on it

Reply:
Its the naming convention we ended up with for our log files, so it won't be unique (but agreed, the names sort of suck).